cristin-project/src/main/java/no/unit/nva/cristin/projects/ProjectPatchValidator.java

```diff
@@ -0,0 +1,141 @@
+package no.unit.nva.cristin.projects;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import no.unit.nva.cristin.projects.model.nva.NvaContributor;
+import nva.commons.apigateway.exceptions.BadRequestException;
+import nva.commons.core.StringUtils;
+
+import java.net.URI;
+import java.time.Instant;
+import java.util.List;
+import java.util.Map;
+
+import static no.unit.nva.cristin.model.Constants.OBJECT_MAPPER;
+import static no.unit.nva.cristin.model.JsonPropertyNames.ACADEMIC_SUMMARY;
+import static no.unit.nva.cristin.model.JsonPropertyNames.ALTERNATIVE_TITLES;
+import static no.unit.nva.cristin.model.JsonPropertyNames.CONTRIBUTORS;
+import static no.unit.nva.cristin.model.JsonPropertyNames.COORDINATING_INSTITUTION;
+import static no.unit.nva.cristin.model.JsonPropertyNames.END_DATE;
+import static no.unit.nva.cristin.model.JsonPropertyNames.LANGUAGE;
+import static no.unit.nva.cristin.model.JsonPropertyNames.POPULAR_SCIENTIFIC_SUMMARY;
+import static no.unit.nva.cristin.model.JsonPropertyNames.START_DATE;
+import static no.unit.nva.cristin.model.JsonPropertyNames.TITLE;
+import static no.unit.nva.language.LanguageConstants.UNDEFINED_LANGUAGE;
+import static no.unit.nva.language.LanguageMapper.getLanguageByIso6391Code;
+import static no.unit.nva.language.LanguageMapper.getLanguageByUri;
+import static nva.commons.core.attempt.Try.attempt;
+
+
+public class ProjectPatchValidator {
```

Comment:
A lot of the code in this class is repeated; I think there is a missed abstraction